packages/client/hooks/useTrebuchetEvents.ts

```diff
@@ -13,6 +13,7 @@ const useTrebuchetEvents = () => {
   const {history} = useRouter()
   const firewallMessageSentRef = useRef(false)
   const recentDisconnectsRef = useRef([] as number[])
+  const serverVersion = useRef(__APP_VERSION__)
```

Comment:
+1 I know I named it `serverVersion`, but in context it should be `serverVersionRef` to be consistent.

Reply:
Fixed